iq-messenger-client/src/app/components/chat-container/chat-container.component.ts

```diff
@@ -1,10 +1,25 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import { RootStoreState } from 'src/app/store';
+import { selectChatState } from 'src/app/store/chat-store/selectors';
 
 @Component({
   selector: 'app-chat-container',
   templateUrl: './chat-container.component.html',
-  styleUrls: ['./chat-container.component.scss']
+  styleUrls: ['./chat-container.component.scss'],
 })
-export class ChatContainerComponent {
+export class ChatContainerComponent implements OnInit {
+  private backgroundUrl = '';
 
+  constructor(private store$: Store<RootStoreState.State>) {}
+
+  ngOnInit() {
+    this.store$.select(selectChatState).subscribe((state) => {
```

Comment:
unsubscribe is required

Reply:
in similar cases my proposal is to use async pipe just in template, as this.backgroundUrl is not required for other tasks